mm2src/lp_ordermatch.rs

```diff
@@ -29,7 +29,7 @@ use common::{bits256, json_dir_entries, now_ms, new_uuid,
   remove_file, rpc_response, rpc_err_response, write, HyRes};
 use common::executor::{spawn, Timer};
 use common::mm_ctx::{from_ctx, MmArc, MmWeak};
-use common::mm_number::{from_dec_to_ratio, from_ratio_to_dec, MmNumber};
+use common::mm_number::{from_dec_to_ratio, from_ratio_to_dec, Fraction, MmNumber};
```

Comment:
It seems to me, the Fraction should be used before from_* in the queue. Or we should include functions before structures?

Reply:
The ordering I have established is small letters first, snake-case next, all caps last.

Reply:
Yea, let's agree for the future on it, I've mixed it somewhere else, will change if noticed.